nearit-ui/src/main/java/com/nearit/ui_bindings/utils/PreRequirementsUtil.java

```diff
@@ -0,0 +1,49 @@
+package com.nearit.ui_bindings.utils;
+
+import android.bluetooth.BluetoothAdapter;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.location.LocationManager;
+import android.support.v4.content.ContextCompat;
+
+import it.near.sdk.logging.NearLog;
+
+/**
+ * Created by Federico Boschini on 28/09/17.
+ */
+
+public class PreRequirementsUtil {
```

Comment:
Add a javadoc comment to the public static methods of this class

Reply:
done